webapp/channels/src/sass/components/_modal.scss

```diff
@@ -257,7 +257,6 @@
     .modal-header {
         display: flex;
         min-height: 76px;
-        align-items: center;
```

Comment:
is it safe to remove this style? I would rather overwrite in the modal it was needed.

Reply:
Yes, its safe. Have tested other modals.

Reply:
thanks for confirming 👍